Core/GDCore/Extensions/Metadata/ExpressionMetadata.h

```diff
@@ -184,6 +184,67 @@ class GD_CORE_API ExpressionMetadata {
     return *this;
   }
 
+  /**
+   * Check if the instruction can be used in layouts or external events.
+   */
+  bool IsReleventForLayoutEvents() const {
+    return releventContext == "Any" || releventContext == "Layout";

```

Comment:
Same question, should we use something like:
```suggestion
  bool IsScopedToLayoutEvents() const {
    return releventScope == "Any" || releventScope == "Layout";

```